### PR TITLE
Add e2e test for disabled rules

### PR DESCRIPTION
**Pull request by ttuffin**
Closes [AAP-9792](https://issues.redhat.com/browse/AAP-9792).

### CODE REVIEW

File: tests/e2e/files/rulebooks/test_disabled_rules.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+- name: Ruleset 1
+  hosts: all
+  sources:
+    - generic:
+        event_delay: 0.5
```
Comment (Alex-Izquierdo) [inline tests/e2e/files/rulebooks/test_disabled_rules.yml:6]: This should be a variable

File: tests/e2e/files/rulebooks/test_disabled_rules.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        payload:
+          - action: "trigger_enabled_rule"
+          - action: "trigger_disabled_rule"
+
+  rules:
+    - name: Enabled rule
+      enabled: true
```
Comment (Alex-Izquierdo) [inline tests/e2e/files/rulebooks/test_disabled_rules.yml:13]: I wonder if it supports variable substitution.

Comment (ttuffin) [reply tests/e2e/files/rulebooks/test_disabled_rules.yml:13]: Good question. The schema doesn't allow it. @mkanoor, what do you think about allowing variable substitution or the `enabled` param?

File: tests/e2e/files/rulebooks/test_disabled_rules.yml (hunk 1 above)
Comment (ttuffin) [reply tests/e2e/files/rulebooks/test_disabled_rules.yml:6]: The default event delay is too high so we won't use that. Do you mean to specify the value in the test, or add a new key in default.yml?

File: tests/e2e/files/rulebooks/test_disabled_rules.yml (hunk 2 above)
Comment (mkanoor) [reply tests/e2e/files/rulebooks/test_disabled_rules.yml:13]: @ttuffin I am not sure about it, so far only the name has substitution enabled. We can discuss in the dev sync call

Comment (Alex-Izquierdo) [reply tests/e2e/files/rulebooks/test_disabled_rules.yml:13]: I think it could be good. Let's discuss it there.

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by mkanoor — APPROVED** [review]

**Pull request merged**